extension/src/config/config.js

```diff
@@ -17,7 +17,7 @@ function getModuleConfig() {
       : undefined,
     addCommercetoolsLineItems: _getValueOfBooleanFlag(
       config.addCommercetoolsLineItems,
-      true
+      false
```

Comment:
For web component version 5, we would not add lineitems for specific payment methods, because we don't need to specify payment methods in `/session` API call.

To add line items, merchant can do either approaches during create session : 
1. Define line items by themselves in web component to build the create session request
2. Set `addCommercetoolsLineItems` to `true` within `createSessionRequest` so that integration can generate line items for merchants.

If merchants don't do the above approach, the `sessionData` from create session response would not support web component for some payment methods which requires line items, such as affirm and klarna. That means, it would be merchants responsibility to enable line items by themselves if they know affirm and klarna is required to be supported.

I have made a fix since we didn't support `addCommercetoolsLineItems` for create session request.
https://github.com/commercetools/commercetools-adyen-integration/pull/1061/commits/40fbf55f2eac87ce833bf8968efefa57ea5f2c90


I would also update the documentation in the next branch.

Reply:
Yes, Please make sure the documentation is updated correctly.